aries-site/src/layouts/content/Example/ExampleControls.js

```diff
@@ -78,6 +86,19 @@ export const ExampleControls = ({
             />
           </Box>
         )}
+        {componentCode && (
+          <Box flex={false}>
+            <Button
+              title="View component code"
+              a11yTitle="View component code"
+              href={componentCode}
+              icon={<Code />}
+              label={!horizontalLayout && 'View component code'}
```

Comment:
Opening for discussion. With the this icon and label, are we introducing a point of confusion where folks might think they will see the example's implementation?

Ideas:
- Should we use the Github icon and label similar to Grommet site?
- Should the button label read "View ${componentCode.name} source"?
- Others?
- Does the "Show Code" panel below get relabeled as "Show implementation"? "Show example's code"? ...

Just trying to distinguish between implementation code and component source code...